spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/DataFlowTemplate.java

```diff
@@ -90,6 +90,7 @@ public DataFlowTemplate(URI baseURI, RestTemplate restTemplate) {
 		Link link = getLink(resourceSupport, "tasks");
 		resources.put("tasks/definitions", new UriTemplate(link.getHref() + "/definitions"));
 		resources.put("tasks/deployments", new UriTemplate(link.getHref() + "/deployments"));
+		resources.put("tasks/executions", new UriTemplate(link.getHref() + "/executions"));
```

Comment:
Why do we need to do this for tasks and not for the other endpoints?  Are we doing something wrong here?